app/src/main/java/ch/epfl/sdp/blindly/profile_setup/ProfileGenderMore.kt

```diff
@@ -10,14 +10,20 @@ import ch.epfl.sdp.blindly.R
 private val REGEX = Regex("^[a-zA-Z]*$")
 
 class ProfileGenderMore : AppCompatActivity() {
+
+    private var username: String? = null
+    private var birthday: String? = null
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.profile_setup_gender_more)
+        val bundle = intent.extras
+        username = bundle?.getString(EXTRA_USERNAME)
+        birthday = bundle?.getString(EXTRA_BIRTHDAY)
     }
 
     fun startProfileOrientation(view: View) {
         findViewById<TextView>(R.id.warning1_p4_2).visibility = View.INVISIBLE
-        findViewById<TextView>(R.id.warning2_p4_2).visibility = View.INVISIBLE
```

Comment:
Why this was removed, if regex doesn't match in line 30 warning2_p4_2 is shown. If on the second try an empty text is given as input warning1_p4_2 is shown with line 44. By deleting this line you would make them both stay on page superposed.

Reply:
Yep my mistake! It's back in the next commit :)